### DISCUSSION TITLE
Welcome to ARC-Alkali-Rydberg-Calculator Discussions!

### DISCUSSION THREAD

**Discussion by nikolasibalic** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#91] [Announcements]
## 👋 Welcome to ARC discussions!
  We’re using Discussions as a place to connect with other members of community (AMO physics and beyond, anyone who finds ARC useful) on questions regarding use of ARC.

Any bug reports and pressing issues regarding code development and maintenance should still go directly to GitHub Issues.

Discussions are intended to be an easy way where you can

  * Ask questions you’re wondering about;
  * Share ideas; and
  * Engage with other community members.

**Comment by yadiraarp** [comment]
hello i am having problems using the program, i use anconda on Mac
numpy.core.multiarray failed to import   is the error that marks

**Reply by nikolasibalic** [reply]
Hello @yadiraarp 

Thank you for this report. We had situations like that reported before with Anaconda and Numpy due to outdated Numpy versions.
Could you please check solution provided here: https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/issues/63#issuecomment-829159366 

Let me know if this resolves your problem.
